java-generator/core/src/main/java/io/fabric8/java/generator/nodes/JObject.java

```diff
@@ -314,4 +350,8 @@ static ClassOrInterfaceType toClassOrInterfaceType(String className) {
     return withoutDollars.indexOf('<') >= 0 ? StaticJavaParser.parseClassOrInterfaceType(withoutDollars)
         : new ClassOrInterfaceType(null, withoutDollars);
   }
+
+  private static String sanitizeFieldNameFromDefinition(Map.Entry<String, JSONSchemaProps> fieldDefinition) {
```

Comment:
[nitpick] we are using this method exactly once, I would not do this refactoring ...

Reply:
You're definitely right, thanks, fixing!

Reply:
This is fixed, resolving.